app/controllers/concerns/submission_scoring.rb

```diff
@@ -12,22 +12,25 @@ def collect_test_results(submission)
         output = execute_test_file(file, submission)
         assessment = assessor.assess(output)
         passed = ((assessment[:passed] == assessment[:count]) and (assessment[:score]).positive?)
-        testrun_output = passed ? nil : 'message: ' + output[:message].to_s + "\n stdout: " + output[:stdout].to_s + "\n stderr: " + output[:stderr].to_s
+        testrun_output = passed ? nil : 'status: ' + output[:status].to_s + "\n stdout: " + output[:stdout].to_s + "\n stderr: " + output[:stderr].to_s
```

Comment:
As discussed today, `message` is always `nil`.

Reply:
if this is the case, just omit / remove it.